ycmd/utils.py

```diff
@@ -29,8 +29,10 @@
 import subprocess
 import collections
 
-WIN_PYTHON27_PATH = 'C:\python27\pythonw.exe'
-WIN_PYTHON26_PATH = 'C:\python26\pythonw.exe'
+WIN_PYTHON27_PATH = 'C:\python27\python.exe'
+WIN_PYTHON26_PATH = 'C:\python26\python.exe'
+# Creation flag to disable creating a console window on Windows
```

Comment:
Wouldn't be better to link https://msdn.microsoft.com/en-us/library/windows/desktop/ms684863(v=vs.85).aspx in the comment? so one can figure out where this flag came from without looking at the issue tracker.